image_filters/README.md

```diff
@@ -0,0 +1,44 @@
+# Vello Image Filters
+
+In 2d Rendering, you often need to process images in ways which aren't just drawing of paths.
+For example, by blurring a background, zooming in, or inverting colours.
+This folder provides implementations of several different image filters, using different backends.
+
+These backends are:
+
+- The "reference" CPU backend, found in [`vello_filters_cpu`].
+  We intend to publish this, and use it in Vello CPU.
+- GPU shader code, without a driver.
+  We haven't yet decided how this will be distributed.
+  This is not yet implemented.
+- Utilities and pipelines for using the GPU shader with wgpu for simple cases.
+  We intend to publish this crate, and might use it in Vello Hybrid.
+  This is not yet implemented.
+
+We intend to have GPU implementations which are able to be executed on WebGL, with more optimised versions (e.g. using compute shaders) for more powerful GPUs.
+
+In this README, we use the "Svg" prefix to refer to filters from the Filter Effects Module Level 1, found at <https://drafts.fxtf.org/filter-effects/>.
+The planned filters are (at least), and the implementations we have working are:
+
+| Filter                              | CPU | GPU (planned) | wgpu (planned) |
+| ----------------------------------- | --- | ------------- | -------------- |
+| [Svg Gaussian Blur][feGaussianBlur] |     |               |                |
```

Comment:
`feOffset` is a candidate for inclusion in an initial version: it's useful for drop shadows and if you're shifting by fractional pixels, you may want may want bi{linear,cubic} filtering.